docs/README.md

```diff
@@ -1,272 +1,278 @@
 ---
+
 title: Metabase documentation
 redirect_from:
-  - /docs/latest/enterprise-guide
-  - /docs/latest/users-guide
-  - /docs/latest/administration-guide
-  - /docs/latest/operations-guide
-  - /docs/latest/faq
+
+* /docs/latest/enterprise-guide
+* /docs/latest/users-guide
+* /docs/latest/administration-guide
+* /docs/latest/operations-guide
+* /docs/latest/faq
+
```

Comment:
This change is not appropriate. This section is Jekyll frontmatter and as such, it must use YAML format.